### PR TITLE
current location before change

### PR DESCRIPTION
**Pull request by eugene0928**

### CODE REVIEW

File: scenes/location.ts [hunk 1]
```diff
@@ -11,7 +11,10 @@ import { IPrayTime, IUser } from '#types/database'
 let scene = new Scene<BotContext>('Location')
 
 scene.step(async (ctx) => {
-  const message = HLanguage('chooseRegion')
+  let currentRegionMsg = HLanguage('currentRegion');
+  currentRegionMsg = HReplace(currentRegionMsg, ['$region'], [ctx.user.region])
```
Comment (Xayrulloh) [inline scenes/location.ts:15]: would u agree if just added current location after msg instead of replacing

**Review by Xayrulloh — CHANGES_REQUESTED** [review]

File: translate/uz.json [hunk 2]
```diff
@@ -90,7 +90,7 @@
     "Chortoq": 35,
     "Chust": 33
   },
-  "chooseRegion": "Shaxaringizni belgilang",
+  "chooseRegion": "Shahringizni belgilang",
```
Comment (toshmukhamedov) [inline translate/uz.json:93]:
```suggestion
  "chooseRegion": "Mintaqani belgilang",
```
Chunki hammasi regionladgi hammasi shahar emas manimcha

File: translate/uz.json [hunk 3]
```diff
@@ -147,5 +147,6 @@
   "sourceMessage": "_Namoz vaqtlari:_ [islom.uz]($siteLink)\n_Hadislar to'plami:_ [Riyozus solihiyn]($sourceLink)",
   "share_qu'ron_va_tafsiri": "https://t.me/Quron_va_Tafsiri\n\nAssalomu alaykum. Ushbu kanalda siz qur'on va uning tafsirini audio tarzda eshitib o'rganishingiz mumkin. Audio format avval o'zbek tilida keyin esa arab tilida keltirilgan. O'ylaymizki bu siz uchun manfatli bo'ladi",
   "feedbackStartMessage": "Taklif yoki shikoyatingizni yozing",
-  "feedbackEndMessage": "Taklif yoki shikoyatingizni yozganingiz uchun rahmat"
+  "feedbackEndMessage": "Taklif yoki shikoyatingizni yozganingiz uchun rahmat",
+  "currentRegion": "Sizning hozirgi shahringiz: <i>$region</i>"
```
Comment (toshmukhamedov) [inline translate/uz.json:151]:
```suggestion
  "currentRegion": "Hozirgi mintaqa: <i>$region</i>"
```

File: scenes/location.ts (hunk 1 above)
Comment (eugene0928) [reply scenes/location.ts:15]: for sure

**Review by toshmukhamedov — APPROVED** [review]

**Review by Xayrulloh — APPROVED** [review]

**Pull request merged**